main/src/main/java/org/mobilitydata/gtfsvalidator/report/model/FeedMetadata.java

```diff
@@ -78,10 +82,18 @@ public static FeedMetadata from(GtfsFeedContainer feedContainer, ImmutableSet<St
 
     feedMetadata.setCounts(feedContainer);
 
-    if (feedContainer.getTableForFilename(GtfsFeedInfo.FILENAME).isPresent()) {
```

Comment:
This requires that all 4 files be present, but calendar and calendar_date are conditionally required, meaning one of the two could be absent.